### PR TITLE
init of NSTimer object that does not go through the designated initia…

### PR DESCRIPTION
**Pull request by msft-Jeyaram**
…lizer #772.

### CODE REVIEW

**Comment by ms-jihua** [comment]
:shipit:

File: tests/unittests/Foundation/NSTimerTests.mm [hunk 1]
```diff
@@ -20,10 +20,6 @@
 #import <future>
 #import <windows.h>
 
-TEST(NSTimer, Init) {
-    ASSERT_TRUE_MSG([[[NSTimer alloc] init] autorelease] != nil, "FAILED: alloc/init failed.");
-}
```
Comment (ms-jihua) [inline tests/unittests/Foundation/NSTimerTests.mm]: LOL

**Comment by DHowett-MSFT** [comment]
This test is gold.

**Comment by DHowett-MSFT** [comment]
:shipit:

File: tests/unittests/Foundation/NSTimerTests.mm (hunk 1 above)
Comment (msft-Jeyaram) [inline tests/unittests/Foundation/NSTimerTests.mm]: :joy:

**Comment by msft-Jeyaram** [comment]
[Au] LOL it was the test of all tests.

**Pull request merged**